### PR TITLE
Added billboard for 2024 developer survey

### PR DESCRIPTION
**Pull request by bmispelon**
This was requested by @thibaudcolas .

I took inspiration from a4799dc44be834c6376c07739a222b1a55d6ffd1 for the wording (found using `git log -S survey`).

Here's how it looks (light mode):

Screenshot 2024-11-21 at 21-07-37 Django overview Django

### CODE REVIEW

File: djangoproject/scss/_style.scss [hunk 1]
```diff
@@ -699,6 +699,7 @@ blockquote {
```
Comment (sabderemane) [inline djangoproject/scss/_style.scss:699]: I can't make the suggestion above but could you add a border-top in #billboard style like this?

```suggestion
#billboard {
    ...
    border-top: 1px solid $black;
    ...
```

IMO, it better differentiates with the banner and title above.

How it's look like with it:
Capture d’écran 2024-11-25 à 19 06 21
Capture d’écran 2024-11-25 à 19 06 12

**Review by sabderemane — COMMENTED** [review]
Small suggestion but otherwise seems to look good to me! Thank you Baptiste ⭐

File: djangoproject/scss/_style.scss (hunk 1 above)
Comment (bmispelon) [reply djangoproject/scss/_style.scss:700]: Good idea, I've added the change. Thanks!

**Comment by bmispelon** [comment]
Thanks for the review @sabderemane ! 💚

**Pull request merged**